runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -73,18 +74,36 @@ private int toHourly(int value)
 		return (int) ((1.0 / (getTimeElapsedInSeconds() / 3600.0)) * value);
 	}
 
+	int getXpGained()
+	{
+		return xpGained - xpPaused;
```

Comment:
This is getting too complicated.
I'd rather you do something like (pardon the style difference)

```
if (paused) {
return xpWhenPaused;
}
return xpGained;
```

Reply:
See above.